src/systems/eu/implementations/react/components/select/src/Select.jsx

```diff
@@ -92,7 +94,11 @@ const Select = ({
             : {})}
         >
           {options.map((option) => (
-            <option key={option.label} value={option.value}>
+            <option
+              key={option.label}
+              value={option.value}
+              disabled={option.isDisabled}
```

Comment:
In twig we are not, but with react we are able to spread over additional properties coming from the option.
```
{...option}
```
I personally see a benefit of it. Having key, value, disabled is enough. We'll need to handle "selected" as well.